pkg/hookbot/hookbot.go

```diff
@@ -425,7 +424,17 @@ func (h *Hookbot) ServeSubscribe(conn *websocket.Conn, r *http.Request) {
 		}
 
 		conn.SetWriteDeadline(time.Now().Add(90 * time.Second))
-		err := conn.WriteMessage(websocket.BinaryMessage, message.Body)
+		_, isRecursive := recursive(topic)
+		err := error(nil)
+		if isRecursive {
+			topicMsg := []byte{}
+			topicMsg = append(topicMsg, message.Topic...)
+			topicMsg = append(topicMsg, '\x00')
+			topicMsg = append(topicMsg, message.Body...)
+			err = conn.WriteMessage(websocket.BinaryMessage, topicMsg)
+		} else {
+			err = conn.WriteMessage(websocket.BinaryMessage, message.Body)
+		}
```

Comment:
Nice and simple. Could you refactor this so that there is only one call to `conn.WriteMessage`?